### PR TITLE
ZCA Whiteing demo

### PR DESCRIPTION
**Pull request by marianna13**
#### Changes
Added ZCA Whiteing Gradio demo (https://huggingface.co/spaces/kornia/zca-whiteing/)

#### Type of change

- [ x] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

File: docs/source/enhance.rst [hunk 1]
```diff
@@ -69,6 +69,13 @@ Modules
...
 .. raw:: html
-   :file: gradio/enhance_gradio.html
+   :file: gradio/zca_whitening.html
 
 Visit the demo on `Hugging Face Spaces <https://huggingface.co/spaces/kornia/kornia-image-enhancement>`_.
+
+ZCA Whitening Interactive Demo
+----------------
+.. raw:: html
+   :file: gradio/enhance_gradio.html
```
Comment (edgarriba) [inline docs/source/enhance.rst:79]: @marianna13 check this PR https://github.com/kornia/kornia/pull/1924/files

**Review by edgarriba — CHANGES_REQUESTED** [review]

File: docs/source/enhance.rst (hunk 1 above)
Comment (marianna13) [reply docs/source/enhance.rst]: I updated it :)

File: docs/source/gradio/zca_whitening.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<iframe src="https://hf.space/embed/kornia/zca-whiteing/+"></iframe>
```
Comment (edgarriba) [inline docs/source/gradio/zca_whitening.html:1]: i don;'t think this is needed anymore

File: docs/source/enhance.rst [hunk 3]
```diff
@@ -69,6 +69,11 @@ Modules
...
 .. raw:: html
-   :file: gradio/enhance_gradio.html
+   :file: gradio/zca_whitening.html
 
 Visit the demo on `Hugging Face Spaces <https://huggingface.co/spaces/kornia/kornia-image-enhancement>`_.
+
+ZCA Whitening Interactive Demo
+----------------
+.. raw:: html
+<gradio-app space="kornia/zca-whiteing"></gradio-app>
```
Comment (edgarriba) [inline docs/source/enhance.rst:79]:
```suggestion
  <gradio-app space="kornia/zca-whitening"></gradio-app>
```
you have a typo

File: docs/source/enhance.rst [hunk 4]
```diff
@@ -69,6 +69,11 @@ Modules
 Interactive Demo
 ----------------
 .. raw:: html
-   :file: gradio/enhance_gradio.html
+   :file: gradio/zca_whitening.html
```
Comment (edgarriba) [inline docs/source/enhance.rst:72]:
```suggestion
```
don't remove the existing one

File: docs/source/gradio/zca_whitening.html (hunk 2 above)
Comment (edgarriba) [reply docs/source/gradio/zca_whitening.html:1]: /cc @NimaBoscarino

**Review by edgarriba — COMMENTED** [review]
oh, seems that the type came from hf repo name.

also, i'll some comments there but check other examples, you can fully implement the whole thing using only kornia api: read_image, resize, grid, etc

**Review by edgarriba — APPROVED** [review]

**Comment by edgarriba** [comment]
@marianna13 i'll check into the spaces soon this weekend to see how to fix, but definitely is doable

**Pull request merged**